main/.vuepress/config.js

```diff
@@ -36,42 +36,34 @@ module.exports = {
       'script',
       {},
       `
+    const withSelector = (selector, callback) => {
+      const el = document.querySelector(selector);
+      if (el) {
+        callback(el);
```

Comment:
If you're not using the return value of the callback, and we still want to be robust against callback failure, can I suggest something like:
```suggestion
        Promise.resolve(el).then(callback);
```

Reply:
Yeah, I noticed that as well. Updated to adopt that pattern in the `setInterval` callback and ended up removing `withSelector` altogether, and also restored handling of the home-link logo and text in a single callback to match pre-#840 behavior. The changes are substantial enough to warrant re-review; PTAL.